src/models/WidgetData.ts

```diff
@@ -196,4 +196,4 @@ export interface IWidgetKilledAndInjuredCountPerAgeGroup extends IWidgetDataBase
     subtitle?: string;
     title?: string;
   };
-}
+}
```

Comment:
Here as well

Reply:
It doesn't show spaces in the source code here

Reply:
I think maybe you added or removed a space